src/main/java/frc/team166/robot/subsystems/Manipulator.java

```diff
@@ -124,6 +130,10 @@ protected void end() {
         };
     }
 
+    public Command CubePickupWithLights(int blinkCount) {
```

Comment:
Need to get this working... I would try and work with @msoucy or @bot190 (see if they can figure out what's wrong, since I failed)

Reply:
Consider this fixed, for now - #22 fixes this more thoroughly, but we have a workaround in place